spec/system/sdg_management/relations_spec.rb

```diff
@@ -88,58 +93,215 @@
       expect(page).to have_css "h2", exact_text: "Build a hospital"
     end
 
-    describe "search" do
-      scenario "search by terms" do
-        create(:poll, name: "Internet speech freedom")
-        create(:poll, name: "SDG interest")
+    scenario "list records pending to review for the current model by default" do
+      create(:debate, title: "I'm a debate")
+      create(:sdg_review, relatable: create(:debate, title: "I'm a reviewed debate"))
 
-        visit sdg_management_polls_path
+      visit sdg_management_debates_path
 
-        fill_in "search", with: "speech"
-        click_button "Search"
+      expect(page).to have_text "I'm a debate"
+      expect(page).not_to have_text "I'm a reviewed debate"
 
-        expect(page).to have_content "Internet speech freedom"
-        expect(page).not_to have_content "SDG interest"
-      end
+      click_link "Pending"
 
-      scenario "goal filter" do
-        create(:budget_investment, title: "School", sdg_goals: [SDG::Goal[4]])
-        create(:budget_investment, title: "Hospital", sdg_goals: [SDG::Goal[3]])
+      expect(page).to have_text "I'm a debate"
+      expect(page).not_to have_text "I'm a reviewed debate"
+    end
 
-        visit sdg_management_budget_investments_path
-        select "4. Quality Education", from: "goal_code"
-        click_button "Search"
+    scenario "list all records for the current model when user clicks on 'all' tab" do
+      create(:debate, title: "I'm a debate")
+      create(:sdg_review, relatable: create(:debate, title: "I'm a reviewed debate"))
 
-        expect(page).to have_content "School"
-        expect(page).not_to have_content "Hospital"
-      end
+      visit sdg_management_debates_path
+      click_link "All"
+
+      expect(page).to have_text "I'm a debate"
+      expect(page).to have_text "I'm a reviewed debate"
     end
 
-    scenario "target filter" do
-      create(:budget_investment, title: "School", sdg_targets: [SDG::Target[4.1]])
-      create(:budget_investment, title: "Preschool", sdg_targets: [SDG::Target[4.2]])
+    scenario "list reviewed records for the current model when user clicks on 'reviewed' tab" do
+      create(:debate, title: "I'm a debate")
+      create(:sdg_review, relatable: create(:debate, title: "I'm a reviewed debate"))
+
+      visit sdg_management_debates_path
+      click_link "Marked as reviewed"
 
-      visit sdg_management_budget_investments_path
-      select "4.1", from: "target_code"
-      click_button "Search"
+      expect(page).not_to have_text "I'm a debate"
+      expect(page).to have_text "I'm a reviewed debate"
+    end
 
-      expect(page).to have_content "School"
-      expect(page).not_to have_content "Preschool"
+    describe "search" do
+      describe "on 'pending' tab" do
```

Comment:
Since system specs are very slow, how about adding one case checking the search respects the current filter instead of adding the same cases for each filter?